src/main/java/com/example/notificationservice/utils/FunctionsAndConditions.java

```diff
@@ -0,0 +1,56 @@
+package com.example.notificationservice.utils;
+
+
+import com.example.notificationservice.entity.domain.ElasticSearchSchema;
+import com.example.notificationservice.entity.domain.SmsRequests;
+import com.example.notificationservice.kafkaConsumers.SmsRequestConsumer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+
+@Component
+public class FunctionsAndConditions {
+
+    private static final Logger logger = LoggerFactory.getLogger(FunctionsAndConditions.class);
+    public boolean validatePhoneNumber(String number){
+
+        if(number==null||number.length()!=13||number.charAt(0)!='+'||number.charAt(1)!='9'||number.charAt(2)!='1'){
+            logger.info("Initial condition of phoneNumber is not valid");
+            return false;
+        }
+        for(int i=3;i<number.length();i++){
```

Comment:
what is "i" here ?

Reply:
We always use to have a proper naming convention where name itself suggest what is it and what is the use case